frontend/pkg/frontend/otel_sdk.go

```diff
@@ -36,7 +53,7 @@ func InstallOpenTelemetryTracer(ctx context.Context, logger *slog.Logger, resour
 	if _, isNoop = exp.(*noopSpanExporter); !isNoop || autoexport.IsNoneSpanExporter(exp) {
 		isNoop = true
 	}
-	logger.InfoContext(ctx, "initialising OpenTelemetry tracer", "isNoop", isNoop)
+	logger.Info("initialising OpenTelemetry tracer", "isNoop", isNoop)
```

Comment:
Why did we remove the context?

Reply:
it was for consistency with the rest of code which doesn't call the logger.*Context() methods but I can revert. In the current implementation, the frontend initializes a slog [JSON handler](https://pkg.go.dev/log/slog#JSONHandler) which doesn't make any use of the context.